dl4j-labs/src/main/java/ai/certifai/training/classification/heartDiseaseBinaryClassification.java

```diff
@@ -0,0 +1,212 @@
+
+        // Get total length of data
+        int numTrainData = (int) Math.round(ratioTrainTestSplit * totalData);
+        int idx = 0;
+        while (rr.hasNext()) {
+            if(idx < numTrainData) {
+                rawTrainData.add(rr.next());
+            } else {
+                rawTestData.add(rr.next());
+            }
+            idx++;
+        }
+
+        System.out.println("Total train Data " + rawTrainData.size());
+        System.out.println("Total test Data " + rawTestData.size());
+
+        List<List<Writable>> transformedTrainData = transformData(rawTrainData);
+        List<List<Writable>> transformedTestData = transformData(rawTestData);
+
+        DataSetIterator trainData = makeIterator(transformedTrainData);
+        DataSetIterator testData = makeIterator(transformedTestData);
+
+        DataNormalization normalizer = new NormalizerStandardize();
+        normalizer.fit(trainData);
+        trainData.setPreProcessor(normalizer);
+        testData.setPreProcessor(normalizer);
+
+        //=====================================================================
+        //            Step 2: Define Model
+        //=====================================================================
+
+        MultiLayerConfiguration conf = new NeuralNetConfiguration.Builder()
+                .seed(1234)
+                .updater(new Nesterovs(0.001, Nesterovs.DEFAULT_NESTEROV_MOMENTUM))
+                .weightInit(WeightInit.XAVIER)
+                .list()
+                .layer(0, new DenseLayer.Builder()
+                        .activation(Activation.RELU)
+                        .nIn(13)
+                        .nOut(32)
+                        .build())
+                .layer(1, new DropoutLayer(0.3))
+                .layer(2, new DenseLayer.Builder()
+                        .activation(Activation.RELU)
+                        .nOut(64)
+                        .build())
+                .layer(3, new OutputLayer.Builder(LossFunctions.LossFunction.MCXENT)
+                        .nOut(2)
+                        .activation(Activation.SOFTMAX)
+                        .build())
+                .build();
+
+        MultiLayerNetwork model = new MultiLayerNetwork(conf);
+        model.init();
+
+        //=====================================================================
+        //            Step 3: Set Listener
+        //=====================================================================
+
+        StatsStorage storage = new InMemoryStatsStorage();
+        UIServer server = UIServer.getInstance();
+        server.attach(storage);
+
+        // Set model listeners
+        model.setListeners(new StatsListener(storage, 10));
+
+        //=====================================================================
+        //            Step 4: Train model
+        //=====================================================================
+
+        Evaluation eval;
+        for(int i=0; i < epoch; i++) {
```

Comment:
space before and after operator 

meaning` for(int i = 0; i < epoch; ++i)`

there's a very minor difference between ++i and i++, personal preference is ++i for the mere acceleration, 
you can stick back to i++ if you insist